clients/cloud/java-springboot/README.adoc

```diff
@@ -0,0 +1,172 @@
+= Spring Boot with Java Example Confluent Cloud Client
+
+NOTE: Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud) using the Spring Boot using Java API.
+
+== Prerequisites
+
+* Java 1.8 or higher to run the demo application
+* Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud) cluster
+* Initialize a properties file at `src/main/resources/application-ccloud.properties` with configuration to your Confluent Cloud cluster:
+
+[source,shell]
+.application-ccloud.properties
+----
+❯ cat src/main/resources/application-ccloud.properties
```

Comment:
@gAmUssA all client examples use a common config file as describe in https://github.com/confluentinc/configuration-templates/blob/master/README.md

Which means that this `application-ccloud.properties` file must be auto-generated from that common config file.  The value in doing it this way:

- provides consistency between examples
- enables users to try different code languages using the same config file
- enables us to test